tests/repository_data/uptane_director_snapshot.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
targets_simpler.json isn't part of a complete test repository's data, no; it's for an individual test prodding just targets.json.

The other files are parts of repositories and are prefixed with the repository of which they're a part:
* uptane_director_root.json
* uptane_director_snapshot.json
* ...
* uptane_mainrepo_root.json
* ...

Should they remain (if we decide Uptane gets that kind of privilege - instead of generalizing a bit), we'd want them in folders in repository_data like this:

* repository_data/uptane_director:
   * root.json
   * ...
* repository_data/uptane_imagerepo:
   * root.json
   * ...
* repository_data/repository:  <--- may or may not want to rename to 'simple', 'default', etc.
   * root.json
   * ...
* repository_data/der_repository:
   * root.der
   * ...